### PR TITLE
Bugfix & support for multiple terminals

### PR DESCRIPTION
**Pull request by nberth**
Split and adapt commits of PR #2 so they can be merged.

Supersedes #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +172,12 @@ On Linux you can see the output of the application in Vs Code itself. Add `gdbtt
 ```
 ![GdbTTY](gdbttyvscode.png)
 
+You can also use these options with `gdbtty`: `xterm`, `gnome-terminal`, `konsole` and `xfce4-terminal`.
+
+### Roadmap
```
Comment (ddeclerck) [inline README.md]: The roadmap section was explicitly removed by Emilien in https://github.com/OCamlPro/superbol-vscode-debug/pull/1. Do we want it back ?

**Review by ddeclerck — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -179,8 +172,9 @@ On Linux you can see the output of the application in Vs Code itself. Add `gdbtt
 ```
 ![GdbTTY](gdbttyvscode.png)
 
-### Documentation
+You can also use these options with `gdbtty`: `xterm`, `gnome-terminal`, `konsole` and `xfce4-terminal`.
 
+### Documentation
 For a more in depth documentation please check the [Superbol Documentation](https://ocamlpro.com/superbol/)
```
Comment (nberth) [inline README.md]: @ddeclerck BTW that page is not public. What do you think would be a more appropriate page? Have we got one?

File: README.md (hunk 1 above)
Comment (nberth) [reply README.md]: Thanks for noticing!

File: README.md (hunk 2 above)
Comment (ddeclerck) [reply README.md]: I guess we only have this for now :
https://get-superbol.com/studio/
(which BTW have some dead links at the bottom...)

Comment (nberth) [reply README.md]: Ok… I'll put that one for now.

**Pull request merged**